### PR TITLE
Fixed some bad links

### PR DESCRIPTION
**Pull request by najela**
@arschmitz  spiderJS reports a **403** error at http://www.bleepingcomputer.com/tutorials/windows-command-prompt-introduction/

### Remark
It seems all sites are migrating to HTTPS. As mentioned at https://github.com/jquery/jquery.com/pull/91#issue-63863412, the use of **SSL is encouraged**. I also noticed that all jQuery sites have valid SSL certificates. Why aren't all HTTP requests redirected to use the HTTPS protocol?

**PS: I don't know  the best place to raise this issue**

### CODE REVIEW

**Comment by arthurvr** [comment]
Thanks for the PR!

> spiderJS reports a 403 error at http://www.bleepingcomputer.com/tutorials/windows-command-prompt-introduction/

Looks like they fixed that.

> Why aren't all HTTP requests redirected to use the HTTPS protocol?

I honestly have no idea. Maybe anyone from the infrastructure team can give us an answer? cc @gnarf

File: pages/documentation.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ If you actually want to make the fixes and improvements, then
 you'll need to [fork](https://help.github.com/articles/fork-a-repo) the
 repositories you'd like to work on. Once you've made changes you'd like
 reviewed for integration, submit a [pull
-request](http://help.github.com/send-pull-requests/). However, we recommend
+request](https://help.github.com/send-pull-requests/). However, we recommend
```
Comment (arthurvr) [inline pages/documentation.md]: This change doesn't seem correct. Both the `http` and the `https` version redirect to a third page.

**Comment by gnarf** [comment]
> > Why aren't all HTTP requests redirected to use the HTTPS protocol?
> I honestly have no idea. Maybe anyone from the infrastructure team can give us an answer? cc @gnarf

First of all, none of the jquery sites have login or anything that requires secure protocols other than jquery.org (which does force https)

Secondly, leaving it as http allows for a bit more caching and less procesor across most of the sites.

The only reason I could think of to force it on for content sites is to avoid MITM stuff.

I'd hapilly enable this on the content sites if y'all want to do and share a little research on best practices and decide we should have it on. @jquery/content

**Comment by gnarf** [comment]
Also regarding jquery/jquery.com#91 that is specifically talking about script and image SRC stuff from cdns

**Comment by arthurvr** [comment]
I don't feel like it's needed for us, no. Though you could as well see it as promoting best practises, which I think we should do. Not really sure myself.

**Comment by gnarf** [comment]
We should discuss at next meeting? Solicit feedback on an issue?

**Comment by arthurvr** [comment]
@gnarf On the meeting sounds fine. Dropped it on the notes.

File: pages/documentation.md (hunk 1 above)
Comment (najela) [inline pages/documentation.md]: It appears this change was made after I sent the PR

Comment (arthurvr) [inline pages/documentation.md]: Yep, that's possible. Though, can you fix it please? Thanks.

**Pull request closed without merging**